waggle-dance/src/main/scripts/waggle-dance.sh

```diff
@@ -11,6 +11,6 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 SCRIPT_DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-java -jar $SCRIPT_DIR/../service/waggle-dance-core-latest-exec.jar "$@"
+java -Dlogging.config=$SCRIPT_DIR/../conf/log4j2.xml -jar $SCRIPT_DIR/../service/waggle-dance-core-latest-exec.jar "$@"
```

Comment:
Hmmm, another way do this would be to add a logging section to the README a bit like https://github.com/HotelsDotCom/circus-train/#logging (but describing log4j instead of log4j) then this can be set in the YAML. If we set it in the YAML does it then override the value set here? If so we could keep this for those who haven't set it in the YAML. Also, did it not output everything to the console by default? It used to (see last line under section https://github.com/HotelsDotCom/waggle-dance/#running-on-the-command-line)

Reply:
> Also, did it not output everything to the console by default? It used to (see last line under section https://github.com/HotelsDotCom/waggle-dance/#running-on-the-command-line)

It was, but when we tried to change the logging level it looked like nothing changed. We started seeing debug lines only when we added this explicitly in the script.